### ISSUE TITLE
Using ioloop explicitly durring connect function call.

### ISSUE THREAD

**Issue by sourcepirate** [jucacrispim/mongomotor#3]
Does mongomotor provide a way to connect with an io_loop explicitly. If not its a good to have enhancement. Since it would be very useful  to use mongomotor along with third party async frameworks like sanic.
```python
@app.listener('before_server_start')
def pre_hook(app, loop):
      connect('<mongo_url>', io_loop=loop)
```

**Comment by jucacrispim** [comment]
It does not, but I like the idea. When I have some free time I'll work on it. Or I may accept an PR. :)

**Comment by jucacrispim** [comment]
Hello, @sourcepirate 

I just released mongomotor 0.14.0. In this version I handle the io_loop argument properly. Now you can use connect(..., io_loop=loop).

Can you take a look and check if it works for you?

Thanks!

**Comment by sourcepirate** [comment]
@jucacrispim sure man thanks. I love too.
